src/client/src/Stream.c

```diff
@@ -1112,6 +1117,11 @@ STATUS putFrame(PKinesisVideoStream pKinesisVideoStream, PFrame pFrame)
         pKinesisVideoStream->diagnostics.lastFrameRateTimestamp = currentTime;
     }
 
+    if (retStatus == STATUS_SUCCESS) {
```

Comment:
No need for the if check - if you are here then you've succeeded. This is also what I was referring to in the Client.c comment about setting the timestamp here.

Reply:
Will change